qadi_hypothesis.py

```
@@ -155,9 +155,7 @@ def main() -> None:
     async def main_async():
         try:
             await setup_llm_providers(
-                openai_api_key=os.getenv("OPENAI_API_KEY"),
-                anthropic_api_key=os.getenv("ANTHROPIC_API_KEY"),
-                google_api_key=os.getenv("GOOGLE_API_KEY"),
+                google_api_key=os.getenv("GOOGLE_API_KEY")
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)